tests/plugins/button/manual/label.md

```diff
@@ -0,0 +1,33 @@
+@bender-tags: 4.10.2, 421, bug, editor, button
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, language
+
+## Normal mode
+
+1. Click `Language` button.
+
+### Expected
+
+* **aria-expanded:** true
+* **label:** Set language
+
+### Unexpected
+
+* **aria-expanded:** null
+* **label:** Set language (Selected)
+
+## Screen reader
+
+**Note**: you need to enable screen reader for this test.
+
+1. Click `Language` button.
```

Comment:
As for screen reader tests, please use keyboard instead. This is a more common way to use computer for users with screen readers.

Reply:
It was so obvious to me that I just understood this step as "click with <kbd>Space</kbd>/<kbd>Enter</kbd> key"…